numba_cuda/numba/cuda/lowering.py

```diff
@@ -1684,31 +1683,13 @@ def decref(self, typ, val):
 
 
 class CUDALower(Lower):
-    def _is_shared_array_call(self, fnty):
-        # Check if function type is a cuda.shared.array call
-        if not hasattr(fnty, "typing_key"):
-            return False
-        return fnty.typing_key is cuda.shared.array
-
     def _lower_call_normal(self, fnty, expr, signature):
```

Comment:
I think this function can now be deleted as it entirely defers to `Lower._lower_call_normal()`.